modules/RTC/LocalStream.js

```diff
@@ -121,4 +121,11 @@ LocalStream.prototype.getId = function () {
     return this.stream.getTracks()[0].id;
 };
 
+LocalStream.prototype.stopTracks = function() {
+  var originalStream = this.getOriginalStream();
+  originalStream && originalStream.getTracks().forEach(function(track) {
+    track.stop();
+  })
+};
+
```

Comment:
And we need to check here whether stop method actually exist. As it is not supported in IE.